.github/workflows/validate-pull-requests.yml

```diff
@@ -14,8 +14,19 @@ jobs:
         uses: actions/setup-dotnet@v3.0.3
         with:
           dotnet-version: 7.0.x
+      - name: Check docstool
+        run: |
+           curl https://www.myget.org/F/particular/api/v3/flatcontainer/particular.docstool/index.json >> docstool.version
+           echo "/docstool" >> $env:GITHUB_PATH
```

Comment:
Is this to add the tool directory to the `PATH`? Is that worth a comment?

Reply:
I first tried to use $HOME/.dotnet/tools but somehow couldn't get it to work and then switched to using a dedicated path. I guess if we could use the home tools path there would be nothing need to add to the path but we might be caching more than we want. If we leave it as is adding a comment might be good yes.